packages/riipen-ui/src/components/Tooltip.jsx

```diff
@@ -78,6 +78,11 @@ class Tooltip extends React.Component {
      */
     open: PropTypes.bool,
 
+    /**
+     * Whether the popover should stay open after leaving the content
+     */
```

Comment:
```suggestion
    /**
     * Whether the popover should stay open after leaving the content.
     */
```

The documentation is generated using an npm script and we like to have a period at the end to clean up the docs.